tests/test_test_utils.py

```diff
@@ -305,3 +305,15 @@ def test_noop(self) -> None:
     assert _hello_world_str == text
 
     await client.close()
+
+
+@pytest.mark.parametrize("hostname,expected_host",
```

Comment:
Turns out you can separate params nicely like this:
```suggestion
@pytest.mark.parametrize(("hostname", "expected_host"),
```